decidim-dev/lib/decidim/dev/test/rspec_support/active_job.rb

```diff
@@ -3,11 +3,11 @@
 RSpec.configure do |config|
   config.include ActiveJob::TestHelper
 
-  config.around do |example|
-    perform_enqueued_jobs do
-      example.run
-    end
-  end
+  # config.around do |example|
+  #   perform_enqueued_jobs do
+  #     example.run
+  #   end
+  # end
```

Comment:
will remove it on the next PR.

Reply:
Can we remove it now? I don't like leaving commented code without a good reason (like "This doesn't work, and we acutally need to do it like this other way"). If we want to recover deleted code, then that is git job :)

Reply:
Done in https://github.com/decidim/decidim/pull/14784/commits/1aae7d8083294d3f62dc3fb65edc3ac13829bbb6